pom.xml

```diff
@@ -28,34 +28,42 @@
     <jenkins-test-harness.version>2.49</jenkins-test-harness.version>
 
     <!-- Project Dependencies Configuration -->
-    <analysis-model-api.version>7.0.1</analysis-model-api.version>
-    <analysis-model.version>7.0.1</analysis-model.version>
-    <forensics-api-plugin.version>0.5.0</forensics-api-plugin.version>
-    <git-forensics-plugin.version>0.5.0</git-forensics-plugin.version>
-
     <module.name>${project.groupId}.warnings.ng</module.name>
 
+    <analysis-model-api.version>7.0.2</analysis-model-api.version>
+    <analysis-model.version>7.0.2</analysis-model.version>
+    <forensics-api-plugin.version>0.6.1</forensics-api-plugin.version>
+    <git-forensics-plugin.version>0.6.0</git-forensics-plugin.version>
+
+    <font-awesome-api.version>5.10.1-1-beta1</font-awesome-api.version>
+    <jquery3-api.version>3.4.1-1-beta1</jquery3-api.version>
+    <popper-api.version>1.15.0-1-beta1</popper-api.version>
+    <bootstrap4-api.version>4.1.3-1-beta1</bootstrap4-api.version>
+    <data-tables-api.version>1.10.20-1-beta2-SNAPSHOT</data-tables-api.version>
```

Comment:
Friends do not let friends deploy snapshots. [tip](https://github.com/jenkinsci/incrementals-tools#enabling-incrementals-the-easy-way)

Reply:
Thanks for the tip, I tried that already (https://github.com/jenkinsci/warnings-ng-plugin/pull/261) but the PR still fails because we do not have maven 3.6.2 installed in our CI.

Reply:
> we do not have maven 3.6.2 installed in our CI

In which CI? If you use the `maven` label, you are getting a recent version.

Reply:
In https://ci.jenkins.io/job/Plugins/job/warnings-ng-plugin/job/incrementals/1/consoleFull

```
14:01:49  Waiting for next available executor on ‘maven’
14:03:40  Running on aci-maven-27m84 in /home/jenkins/workspace/_warnings-ng-plugin_incrementals
[...]
14:04:01  Unpacking https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.5.4/apache-maven-3.5.4-bin.zip to /home/jenkins/tools/hudson.tasks.Maven_MavenInstallation/mvn on aci-maven-27m84
```

Reply:
Ah. https://github.com/jenkinsci/warnings-ng-plugin/pull/261#pullrequestreview-318609047